src/view/components/StatisticsModal.js

```diff
@@ -0,0 +1,66 @@
+import Component from './Component.js';
+import { LottoStore } from '../../domain/Lotto.js';
+import parseStatistics from '../../utils/statistics.js';
+
+export default class StatisticsModal extends Component {
+  setUp() {
+    this.statistics = LottoStore.calculateStatistics(this.props.lottoList);
+    this.earningRate = LottoStore.calculateEarningRate(this.props.lottoList);
+  }
+
+  setEvent() {
+    this.addEvent(
+      'submit',
+      '.lotto-store__statistics-dialog-retry-form',
+      this.handleSubmitForm.bind(this)
+    );
+  }
+
+  template() {
+    return `
+      <dialog class='lotto-store__statistics-dialog'>
+        <div class='lotto-store__statistics-dialog-contents'>
+          <form class='lotto-store__statistics-dialog-close-form' method='dialog'>
+            <button class='lotto-store__dialog-close-btn'>X</button>
+          </form>
+          <h2 class='lotto-store__statistics-title'>🏆 당첨 통계 🏆</h2>
+          <div class='lotto-store__statistics-table'>
+            <div class='lotto-store__statistics-label'>
+              <span class='lotto-store__awards-label'>일치 갯수</span>
+              <span class='lotto-store__prize-label'>당첨금</span>
+              <span class='lotto-store__count-label'>당첨 갯수</span>
+            </div>
+            <ul class='lotto-store__statistics'>
+              ${this.getStatisticsTemplate()}
+            </ul>
+          </div>
+          <div class='lotto-store__earning-rate'>당신의 총 수익률은
+            ${this.earningRate}%입니다.
+          </div>
+          <form class='lotto-store__statistics-dialog-retry-form' method="dialog">
+            <button class='lotto-store__retry-btn'>다시 시작하기</button>
+          </form>
+        </div>
+      </dialog>
+    `;
+  }
+
+  handleSubmitForm() {
+    setTimeout(this.props.initState, 0);
+  }
```

Comment:
이 함수의 역할은 뭘까요??

Reply:
`Form submission canceled because the form is not connected` 문제점을 해결해 보기 위해
또 다른 방법으로 setState 함수를 비동기로 실행시켜 보았습니다
근본적인 결함을 해결하려면 render 함수를 비동기로 바꿔야 한다는 생각이 드는데, 아직까지 성공적으로 적용을 못한 상태입니다😭